src/einsteinpy/symbolic/ricci.py

```diff
@@ -83,7 +92,41 @@ def from_metric(cls, metric):
 
         """
         rc = ChristoffelSymbols.from_metric(metric)
-        return cls.from_christoffels(rc)
+        return cls.from_christoffels(rc, parent_metric=None)
+
+    def change_config(self, newconfig="llll", metric=None):
```

Comment:
`newconfig="ul"`
Ricci tensor is order 2 tensor. It can't have four indices.

Reply:
Resolve this!

Reply:
It was already changed